### ISSUE TITLE
Freeze toaster on mouse over

### ISSUE THREAD

**Issue by enrico-padovani** [Federerer/Notifications.Wpf#7]
Hi there,

Thanks for this nice and tiny library. It' will be really nice to have a feature that pauses the toast from being closed when mouse over. This is especially helpful while reading long error messages and the user need more time.

**Comment by udayteja96** [comment]
@enrico-padovani you have any solution for this ?

**Comment by uday-teja** [comment]
@enrico-padovani you find any solution for this ?

**Comment by enrico-padovani** [comment]
Yes I've ended up using this other similar library: https://github.com/rafallopatka/ToastNotifications which has this feature out of the box.

**Comment by enrico-padovani** [comment]
No, there are alternatives that might work for you, ex: https://github.com/rafallopatka/ToastNotifications

**Comment by uday-teja** [comment]
Ok, i will look in to that thank you :)
